packages/trackerless-network/src/logic/StreamNodeServer.ts

```diff
@@ -22,8 +27,9 @@ export class StreamNodeServer implements INetworkRpc {
     }
 
     async sendData(message: StreamMessage, context: ServerCallContext): Promise<Empty> {
+        const previousPeer = keyFromPeerDescriptor((context as DhtCallContext).incomingSourceDescriptor!)
+        this.config.markForInspection(previousPeer, message.messageRef!)
```

Comment:
Are we able to deliver the info about the message to Inspector via standard `this.config.broadcast` callback?

Reply:
Not really since `broadcast` is only called on non-duplicate messages